Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -69,19 +65,19 @@ noncomputable def repr : ONote → Ordinal.{0}
   | 0 => 0
   | oadd e n a => ω ^ repr e * n + repr a
 
-/-- Auxiliary definition to print an ordinal notation -/
-def toStringAux1 (e : ONote) (n : ℕ) (s : String) : String :=
+/-- Prints `ω^s*n`, omitting `s` if `e = 0` or `e = 1`, and omitting `n` if `n = 1`. -/
```

Comment:
I would use present tense ("Print ...") and omit the full stop.
But thanks for improving this docstring; it is significantly more helpful now!